corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -305,6 +305,22 @@
                         </select>
                     </div>
                 </div>
+
+                {% if app.grid_display_for_some_modules %}
+                <div class="form-group">
+                    <label class="col-sm-2 control-label">
+                        {% trans "Display Style" %}
```

Comment:
Will check with Amelia how the user can be best guided on how to use this. Sample images might be good for Customisation for each module.